src/number_format.js

```diff
@@ -231,7 +182,7 @@ class NumberFormat extends React.Component {
         We are also setting it without timeout so that in normal browser we don't see the flickering
        */
       this.setCaretPosition(el, cursorPos);
-      setTimeout(() => this.setCaretPosition(el, cursorPos), 0);
+      // setTimeout(() => this.setCaretPosition(el, cursorPos), 0);
```

Comment:
As we see when type in mobile chrome it cause to move cursor forward when type invalid charactors